style-editor-customizer.scss

```diff
@@ -10,4 +10,4 @@ NOTE: This file is automatically populated with additional styles if the user se
 
 /** === Non-Latin font fallbacks === */
 
-@include non-latin-fonts( '.wp-block' );
\ No newline at end of file
+@include non-latin-fonts( '.wp-block .mce-content-body' );
```

Comment:
This isn't quite working when I test — blocks aren't inheriting the correct styles until I click on them. 

I think blocks don't actually get the `.mce-content-body` class until they've been active once. You may need to also specify `.editor-rich-text__tinymce` to get that working. You'll also need to target `editor-post-title__input`, since that doesn't get that class either. 😕